scripts/start-app.js

```diff
@@ -0,0 +1,10 @@
+import {server as app} from "./build/index.js"
+
+function shutdownGracefully() {
+    console.log("Server doing graceful shutdown");
+    app.server.close();
```

Comment:
@michael Is there anything we should run to make sure the database connection is gracefully closed here?